content/source/docs/cloud/api/index.html.md

```diff
@@ -113,14 +123,14 @@ For example, if the API endpoint documentation defines the path `/runs` then the
 
 ## JSON API Formatting
 
-The Terraform Cloud endpoints use the [JSON API specification](http://jsonapi.org/), which specifies key aspects of the API. Most notably:
+The Terraform Cloud endpoints use the [JSON API specification](https://jsonapi.org/), which specifies key aspects of the API. Most notably:
```

Comment:
Ah, good catches, thank you. 🙏

Reply:
I give the link checker full credit on these - jsonapi.org returns 301s for http urls now 😆